### PR TITLE
ECF-RP-521: Copy updates

### PR DESCRIPTION
**Pull request by ethanmills**
### Context
Checklist on ticket. Copy updates after review with Shani and Mehmet

### Review Checks
- [x] All pages have automated accessibility checks via cypress
- [x] All `School` queries are correctly scoped - `eligible` when they need to be

### CODE REVIEW

**Comment by ethanmills** [comment]
[Copy updates](https://trello.com/c/OGsJMaXb/521-copy-updates)

File: app/views/nominations/nominate_induction_coordinator/start.html.erb [hunk 1]
```diff
@@ -1,14 +1,20 @@
-<h1>Nominate an induction tutor</h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Nominate an induction lead or tutor</h1>
 
-<%= render partial: 'nominations/school_details', locals: { school: @nominate_induction_tutor_form.school, title: "Your school" } %>
```
Comment (kicferk1) [inline app/views/nominations/nominate_induction_coordinator/start.html.erb:3]: Is this partial still used?

File: app/views/nominations/request_nomination_invite/choose_location.html.erb [hunk 2]
```diff
@@ -1,27 +1,37 @@
...
-          label: { text: "School location", class: "govuk-label govuk-label--m" },
-          hint: { text: "Enter local authority"},
-          options: { include_blank: true },
-          class: ["js-location-select"]) %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Nominate an induction lead or tutor</h1>
+    <p>Your induction lead or tutor will use our online service to manage their statutory induction programme and
+      receive
+      any notifications.</p>
```
Comment (kicferk1) [inline app/views/nominations/request_nomination_invite/choose_location.html.erb:9]: That line breaks look wrong

File: app/views/nominations/nominate_induction_coordinator/start.html.erb (hunk 1 above)
Comment (ethanmills) [reply app/views/nominations/nominate_induction_coordinator/start.html.erb:3]: Yes, in a review screen. Adding another use

**Review by kicferk1 — APPROVED** [review]

**Pull request merged**